day_7/10_little_monkeys.rb

```diff
@@ -0,0 +1,12 @@
+monkey = ["Ten", "Nine", "Eight", "Seven", "Six", "Five","Four", "Three", "Two", "One"]
+monkey.each do |ele|
```

Comment:
Then here you could do `monkeys.each do |monkey|` which I think would be more readable

Reply:
Ok. Thanks.